engine-bash/src/main/kotlin/io/saagie/updatarium/model/action/BashScriptAction.kt

```diff
@@ -36,6 +37,14 @@ fun ChangeSetDsl.bashAction(
     }
 }
 
+fun ChangeSetDsl.bashAction(
+    workingDir: String = ".",
+    timeout: Duration = Duration.ofMinutes(1),
+    file: File
```

Comment:
IMO could be the first argument

Reply:
Question: the `workingDir` may be by default on the file dir ?

Reply:
We'll move it to the first argument :+1: 
The working dir is the directory where the script is run and may be different from the one where the script is located

Reply:
done